### ISSUE TITLE
Compilation Error with libtiff's CMath::CMath dependency.

### ISSUE THREAD

**Issue by blowekamp** [InsightSoftwareConsortium/ITK#4987]
Building SimpleITK against an installed ITK from the master branch has the following error:
```
 The link interface of target "itktiff" contains:

    CMath::CMath

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
  ```

https://github.com/SimpleITK/SimpleITK/actions/runs/11990814469/job/33428736685
https://open.cdash.org/viewBuildError.php?buildid=10045709

### Actual behavior

```none
```

**Comment by blowekamp** [comment]
It look like libtiff defines a custom imported target:
https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/ThirdParty/TIFF/src/itktiff/cmake/FindCMath.cmake

@bradking  Would you agree the best way to address this for ITK is to update libtiff to use "CMath_INCLUDE_DIRS" and "CMath_LIBRARIES" over the imported target?

**Comment by bradking** [comment]
@blowekamp yes, that's the simplest solution.  I don't know why TIFF has a find module for a standard C library component.

**Comment by SimonRit** [comment]
Thanks, the fix did work for RTK, see [here](https://my.cdash.org/viewBuildError.php?type=0&buildid=2734275) and [here](https://my.cdash.org/viewBuildError.php?type=0&buildid=2735666).

**Comment by dzenanz** [comment]
Should this be closed now?

**Comment by dzenanz** [comment]
@blowekamp close this?

**Comment by blowekamp** [comment]
closed by #4989
